packages/docs-app/package.json

```diff
@@ -21,11 +21,13 @@
         "@blueprintjs/docs-data": "^3.37.0",
         "@blueprintjs/docs-theme": "^3.9.1",
         "@blueprintjs/icons": "^3.24.0",
+        "@blueprintjs/popover2": "^0.1.0",
         "@blueprintjs/select": "^3.15.1",
         "@blueprintjs/table": "^3.8.19",
         "@blueprintjs/test-commons": "^0.10.8",
         "@blueprintjs/timezone": "^3.8.1",
         "@documentalist/client": "~3.0.0",
+        "@popperjs/core": "^2.6.0",
```

Comment:
we should just re-export `placements` from `@popperjs/core` so that consumers like docs-app don't need to depend on this library directly.